app/views/main.scala.html

```diff
@@ -135,8 +135,8 @@
 	                  <a href="http://lobid.org/warranty">Gewährleistung</a> | 
                     <a href="http://www.hbz-nrw.de/impressum">Impressum</a> | 
                     <a href="https://github.com/hbz/lobid/blob/master/conf/Datenschutzerklaerung_lobid.textile">Datenschutz</a> | 
-                    <a href="http://twitter.com/lobidorg"><i class="fa fa-twitter" aria-hidden="true"></i> Twitter</a>&nbsp; 
-                    <a href="http://github.com/hbz/lobid-gnd"><i class="fa fa-github" aria-hidden="true"></i> GitHub</a>&nbsp; 
+                    <a href="https://openbiblio.social/@@lobid"><i class="fa-brands fa-mastodon" aria-hidden="true"></i> Mastodon</a>&nbsp;
```

Comment:
Is the double `@@` correct?

Reply:
Yes, it's the way to escape something in scala.